spec

```diff
@@ -665,12 +665,13 @@ beginseg
     include "build/data/code/code_801A5BD0.bss.o"
     include "build/src/code/code_801A7B10.o"
     include "build/data/code/code_801A7B10.bss.o"
-    include "build/src/code/code_801AA020.o"
-    include "build/data/code/code_801AA020.bss.o"
+    include "build/data/code/audio_rodata.rodata.o"
```

Comment:
fyi: I was able to work out that this audio rodata is `audio_init_params.c` from OoT. Thanks for identifying that this is audio-related in this PR, I might have missed it otherwise :)
https://github.com/zeldaret/oot/blob/master/src/code/audio_init_params.c